src/ngx_pagespeed.cc

```diff
@@ -1988,9 +1981,22 @@ void ps_send_to_pagespeed(ngx_http_request_t* r,
                           ngx_chain_t* in) {
   ngx_chain_t* cur;
   int last_buf = 0;
+
   for (cur = in; cur != NULL; cur = cur->next) {
     last_buf = cur->buf->last_buf;
 
+    if (cur->buf->flush && ctx->follow_flushes) {
```

Comment:
Perhaps when you need very precise control in chunkiness (for example, streaming / bandwidth throttling)?
This tries to reflect what the upstream wants us to do the best we can.  If the upstream is too chatty, disabling follow_flushes would make sense (this would argue for defaulting that to disabled). In that case, `flush_html` would be a good to enable?